AnaProd/anaTupleProducer.py

```diff
@@ -56,15 +58,14 @@ def createAnatuple(inFile, treeName, outDir, setup, sample_name, anaCache, snaps
         lumiFilter = LumiFilter(setup.global_params['lumiFile'])
         df = lumiFilter.filter(df)
 
-
     df = df.Define("sample_type", f"static_cast<int>(SampleType::{sample_config['sampleType']})")
-    df = df.Define("sample_name", f"static_cast<int>({zlib.crc32(sample_name.encode())})")
     isSignal = sample_config['sampleType'] in setup.global_params['signal_types']
     applyTriggerFilter = sample_config.get('applyTriggerFilter', True)
     df = df.Define("period", f"static_cast<int>(Period::{period})")
-    df = df.Define("X_mass", f"static_cast<int>({mass})")
-    df = df.Define("X_spin", f"static_cast<int>({spin})")
-    df = df.Define("entryIndex", "static_cast<int>(rdfentry_)")
+    df = df.Define("X_mass", f"static_cast<int>({mass})") # this has to be moved in specific analyses def
+    df = df.Define("X_spin", f"static_cast<int>({spin})") # this has to be moved in specific analyses def
+    df = df.Define("FullEventId", f"""eventId::computeFullEventId(static_cast<ULong64_t>({Utilities.crc16(sample_name.encode())}), static_cast<ULong64_t>({Utilities.crc16(inFile.encode())}), rdfentry_)""")
```

Comment:
```suggestion
    df = df.Define("FullEventId", f"""eventId::computeFullEventId({Utilities.crc16(sample_name.encode())}, {Utilities.crc16(inFile.encode())}, rdfentry_)""")
```
I don't think that `static_cast`s are necessary.